rakelib/utils.rake

```diff
@@ -68,9 +68,11 @@ class Utils
 
   # select the xcode version we want/support
   def self.version_select
-    xcodes = `mdfind "kMDItemCFBundleIdentifier = 'com.apple.dt.Xcode' && kMDItemVersion = '8.*'"`.chomp.split("\n")
+    version = '8.*'
+    Rake.sh 'mdimport /Applications/Xcode* && sleep 1'
```

Comment:
Sorry, I forgot to remove here.
It's fine in [StencilSwiftKit](https://github.com/SwiftGen/StencilSwiftKit/blob/master/rakelib/utils.rake#L72).